### ISSUE TITLE
Running hy2Foam with 5 neutral species and 19 reactions

### ISSUE THREAD

**Issue by iainsymthe** [vincentcasseau/hyStrath#30]
I am trying to run hy2Foam with all file as as in the example case cylinderReactingMach20 but with the shocktube example.  This case works fine with two species N2 and N and the associated reactions in hTCReactionsQK/thermoDEM (default setting for cylinderReactingMach20).  When I try to run the same case with all the neutral species on all 19 reactions, the code crashes immediately (see part of output file).   I have the 0/ directory set-up to include O2, O, NO species and include/initialConditions has  Y_N2 = 0.77, Y_O2=0.23.  Are there any other changes to be made?

===== part of output file
Selecting thermodynamics package 
{
    type            heRho2Thermo;
    mixture         reacting2Mixture;
    transport       BlottnerEucken;
    thermo          decoupledEnergyModes;
    energy          sensible2InternalEnergy;
    equationOfState perfect2Gas;
    specie          advancedSpecie;
}

Selecting chemistry2Reader foam2ChemistryReader
chemistry2Model: Number of species = 5 and reactions = 19
    using integrated reaction rate
#0  Foam::error::printStack(Foam::Ostream&) at ??:?
#1  Foam::sigFpe::sigHandler(int) at ??:?
#2  ? in "/lib64/libc.so.6"
#3  ? at rho2ReactionThermos.C:?
#4  Foam::species::multiThermo<Foam::decoupledEnergyModesThermo<Foam::perfect2Gas<Foam::advancedSpecie> >, Foam::sensible2InternalEnergy>::evels(double, double) const at ??:?
#5  Foam::Specie2Mixture<Foam::reacting2Mixture<Foam::BlottnerEuckenTransport<Foam::species::multiThermo<Foam::decoupledEnergyModesThermo<Foam::perfect2Gas<Foam::advancedSpecie> >, Foam::sensible2InternalEnergy> > > >::HEvel(int, double, double) const at ??:?
#6  Foam::rho2ReactionThermo::initialiseLight() at ??:?
#7  Foam::rho2ReactionThermo::initialise2T_Light() at ??:?
#8  ? at ??:?
#9  ? at ??:?
#10  __libc_start_main in "/lib64/libc.so.6"
#11  ? at ??:?
Floating point exception (core dumped)

**Comment by iainsymthe** [comment]
I was able to run the case.  Must have been some typo in some file.

**Comment by tanmay8547** [comment]
Hi iain
@iainsymthe Were you able to resolve the issue you had in the above simulation?

**Comment by iainsymthe** [comment]
I had some typos in one of the files so the case was stopping immediately. Once I fixed it, I could get it to start (which is what I reported when I closed the issue).  I just ran a few steps and stopped. Are you trying the same problem?  How long did your simulations take to run?
    On Wednesday, November 18, 2020, 08:04:30 AM CST, tanmay singhal <notifications@github.com> wrote:  
 
 


Hi iain
@iainsymthe Were you able to resolve the issue you had in the above simulation?
